ovos_plugin_manager/tts.py

```diff
@@ -139,7 +139,8 @@ def create(config=None):
             else:
                 raise FileNotFoundError("unknown plugin")
         except Exception:
-            LOG.exception('The selected TTS plugin could not be loaded.')
+            modules = ",".join(OVOSTTSFactory.MAPPINGS.keys())
```

Comment:
these are only backwards compat mappings for mycroft.conf files that predate plugin adoption, these map to the old hardcoded integrations

you should instead use https://github.com/OpenVoiceOS/ovos-plugin-manager/blob/dev/ovos_plugin_manager/tts.py#L7

Reply:
ok updated